Rakefile

```diff
@@ -139,6 +147,11 @@ begin
       title 'Running examples'
       Rake::Task['examples:build'].invoke
 
+      if ENV['TRAVIS_PULL_REQUEST'] == 'false'
```

Comment:
We should at least set this to "true" for this branch until we have determined that this PR doesn't break, otherwise we have to fix it on master later.